apps/system-e2e/project.json

```diff
@@ -5,6 +5,25 @@
   "prefix": "system-e2e",
   "generators": {},
   "targets": {
+    "build": {
+      "builder": "@nrwl/node:webpack",
+      "options": {
+        "outputPath": "dist/apps/system-e2e",
+        "main": "apps/system-e2e/src/does-not-matter-one-bit.ts",
+        "tsConfig": "apps/system-e2e/tsconfig.json",
+        "webpackConfig": "apps/system-e2e/webpack.config.js",
```

Comment:
Configuration can be written in TypeScript ([docs](https://webpack.js.org/configuration/configuration-languages/#typescript)). Is it too much effort to do so?

Reply:
It is not the effort but rather that it needs to be in js form at the time of the execution I think. Otherwise it was in TS but I explicitly converted it to JS.

Reply:
I converted this PR to be "draft" since this is still being developed, this might become TS in the end.